tools/_python_framework/Readme.md

```diff
@@ -33,14 +26,36 @@ sudo python mooltipass_tool.py
 - Select the 'vid:16d0 pid:09a0 rev:0100 mi:00' device, click 'Install'
 - Close the window
 
-## Arch Linux
+## Installation on Linux
+
+To access the device as normal user make sure you have set up the [udev rules correct](https://www.themooltipass.com/udev_rule.txt).
```

Comment:
The last paragraph in my PR. Why you added this dependency and what else "did not work" for you with the older instructions.